spark/src/main/kotlin/com/adevinta/spark/tools/modifiers/Conditions.kt

```diff
@@ -39,6 +39,21 @@ import androidx.compose.ui.Modifier
 public inline fun Modifier.ifTrue(predicate: Boolean, builder: () -> Modifier): Modifier =
     then(if (predicate) builder() else Modifier)
 
+/**
+ * Modifier to make it easy to conditionally add a modifier based on [predicate]
+ * ```kotlin
+ * Box(
+ *  modifier = Modifier.fillMaxWidth()
+ *    .ifFalse(isError) { background(SparkTheme.colors.valid) }
+ *    .padding(16.dp)
+ *  ) {...}
+ * ```
+ * @param predicate condition to decide if the [builder] is added or not to the modifier chain
+ * @param builder the modifier(s) to apply when [predicate] is true
+ */
+public inline fun Modifier.ifFalse(predicate: Boolean, builder: () -> Modifier): Modifier =
+    then(if (predicate) builder() else Modifier)
```

Comment:
```suggestion
    then(if (!predicate) builder() else Modifier)
```

Reply:
The javadoc needs an update as well 🙈

Reply:
Done in [cac202e](https://github.com/adevinta/spark-android/pull/76/commits/cac202e2bd3acb6d408b5f8a5669504a05cc6e1f)